AntiSpam/Guild.py

```diff
@@ -218,3 +218,23 @@ def users(self, value):
                 raise DuplicateObject
 
         self._users.append(value)
+        
+    def updateInGuildState(self, userid):
+        """
+        
+        Gets the userobj and updates the state
+        ======
+        userid
+         the id of the user where the state should be updated
+        """
+        user = User(
+            self._bot,
+            userid,
+            self.id,
+            self.options,
+            logger=self.logger,
+        )
+        
+        for userObj in self.users:
```

Comment:
Given this method is only being called once a user joins, can you please ensure you only set `in_guild` to `True`. By simply asking the program to flip the currently stored value I can forsee issues rising with regard to misfired events, the bot simply going offline and a couple other things

Reply:
Done although it gets set to False because now because if we cant create a user obj the user is obviously still in the guild and on_member_leave it then gets set to False.

Reply:
An interesting choice, I may change this after the merge.